misc/scripts/build.sh

```diff
@@ -58,14 +58,14 @@ then
    exit 2
 fi
 
-min_ver=`echo $ver | grep -o '\.[0-9]*\.' | sed 's/\.//g'`
-if [ $min_ver -lt 8 ]
+maj_ver=`echo $ver | sed 's/\..*$//'`
+min_ver=`echo $ver | sed 's/[0-9]*\.//' | sed 's/\..*$//'`
+if [ $maj_ver -lt 17 -a $min_ver -lt 8 ]
```

Comment:
yes, this condition means 1.7 and below :-). See description - yes, we can drop it and document or we can leave it and not document - my (paranoid) preference is to leave it to handle the case and not touch any docs.